applications/MeshingApplication/custom_processes/multiscale_refining_process.cpp

```diff
@@ -391,7 +391,7 @@ void MultiscaleRefiningProcess::CloneNodesToRefine(IndexType& rNodeId)
 
     // Adding the nodes to the refined sub model parts
     StringVectorType sub_model_part_names = mrCoarseModelPart.GetSubModelPartNames();
-    for (auto name : sub_model_part_names)
+    for (const auto& name : sub_model_part_names)
```

Comment:
FYI C++17 added string_view, maybe useful in such cases 
Not sure as I haven't used it myself yet